staging/publishing/rules.yaml

```diff
@@ -1080,189 +1080,193 @@ rules:
     - repository: client-go
       branch: release-1.18
 
-- destination: kube-controller-manager
+- destination: cloud-provider
```

Comment:
This is an unfortunate diff, but the verify script was complaining that cloud-provider was reference after kube-controller-manager since kube-controller-manager now depends on cloud-provider